src/ardrive.ts

```diff
@@ -1122,7 +1176,9 @@ export class ArDrive extends ArDriveAnonymous {
 		const createDriveResult = await this.arFsDao.createPublicDrive(
 			driveName,
 			driveRewardSettings,
-			rootFolderRewardSettings
+			rootFolderRewardSettings,
+			// No wallet owner assertion needed on public drive creations
```

Comment:
Nit: language here is open to interpretation:
`No wallet owner assertion needed` could mean that we need to add an assertion that we don't yet have OR that there's no need to assert it. Since it's the latter, I think it's clearer to say:
`There is no need to assert ownership during drive creation`

Reply:
8664ba6